### PR TITLE
Adding creating an event feature

### PR DESCRIPTION
**Pull request by delekta**
Demo

https://github.com/war-in/suu-nft-project/assets/57197926/56da91ed-e616-4053-a32f-08a2a39ff01a

### CODE REVIEW

**Comment by war-in** [comment]
Looks cool 🚀 Just few comments
- [ ] In the event view, one of the options should be named `Open sale`, because it is available for everyone
- [ ] Rank names should be the same as real ones
- [ ] In the create event view, there should be rank names above each `Sale start time`

File: frontend/src/screens/AdminEventsPanel.tsx [hunk 1]
```diff
@@ -24,53 +24,92 @@ export interface Event {
...
+        maxTicketsPerUserPerRank[index] = +value;
+      } else if (key.includes("ticketPrice")) {
+        ticketPricePerRank[index] = value;
+      }
+    });
+
+    const event: CreateEventRequest = {
+      ticketName: values.name,
+      ticketSymbol: values.symbol,
+      ranksAddress: ranksAddress!, // I shouldnt do it, but ...
```
Comment (t0ms0n00) [inline frontend/src/screens/AdminEventsPanel.tsx:71]: assuming that the rank group exists and it is required in the form I think that `const [ranksAddress, setRanksAddress] = useState("");` should work fine. numberOfRanks hides submit button so suggested approach should be safe

File: frontend/src/screens/AdminEventsPanel.tsx [hunk 2]
```diff
@@ -24,53 +24,92 @@ export interface Event {
 
 function AdminEventsPanel() {
   const [ranksGroups, setRanksGroups] = useState<string[]>();
+  const [numberOfRanks, setNumberOfRanks] = useState<number>(0);
```
Comment (t0ms0n00) [inline frontend/src/screens/AdminEventsPanel.tsx:27]:
```suggestion
  const [numberOfRanks, setNumberOfRanks] = useState(0);
```

**Review by t0ms0n00 — COMMENTED** [review]
👍

**Pull request merged**